### DISCUSSION TITLE
(Crash loop) when trying to run OOBE

### DISCUSSION THREAD

**Discussion by KristalAndStarlight** [valinet/ExplorerPatcher#2485] [General]
with the new changes upcoming with the EU consumer laws, some users might want to change their region, however, with Explorer patcher installed, the oobe just keeps crashing over and over. (may or may not be talking from experience because it happened today)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by KristalAndStarlight** [reply]
not sure how to do that since I don't have access to windows due to crash looping, doesn't let me boot into safe mode either.

**Reply by KristalAndStarlight** [reply]
alright got to it, explorerpatcher is version 22621.2506.60.1 and windows is build 22621.2715

**Comment by Amrsatrio** [comment]
How does one trigger the OOBE? Probably will try to fix it later when time allows.

**Reply by KristalAndStarlight** [reply]
%WINDIR%\system32\sysprep\sysprep.exe
found a way to get out of the crash loop though, simply don't name the device, not sure why that works?

**Reply by pyrates999** [reply]
what do you mean don't name this device?

**Reply by KristalAndStarlight** [reply]
so in the oobe, there's a part where you get to name the device, when we've inputted the name we wanted, it just crashed, when we skipped it, it worked fine

**Reply by pyrates999** [reply]
You can rename the device by opening settings, go to system, and click on about on the left side.
